### DISCUSSION TITLE
A Beginner’s Contribution for the William Alligator Indicator?

### DISCUSSION THREAD

**Discussion by ghost** [DaveSkender/Stock.Indicators#328] [Ideas]
Hello @DaveSkender ,

I discovered your project recently, and because of my interest in it, I wish I could contribute to it.

I wanted to ask you if it was possible for me to work on the integration of William Alligator or was it already planned for you or someone else to do it?

Waiting for your answer.
Cordially,

**Comment by DaveSkender** [comment]
Yes, please do, and thank you for contributing.  Williams Alligator was in the backlog, but we had not started on it yet -- you are welcome to take that one on.  Please review the [contributing guidelines](https://daveskender.github.io/Stock.Indicators/docs/CONTRIBUTING.html#content).  The easiest way to get started is to copy a similar existing `indicators` folder as a starting point.  If you have questions along the way you can post them in this conversation thread or send a direct message [@daveskender](https://twitter.com/messages/compose?recipient_id=27475431).  I'm open to pair programming too if you're interested and want help getting acquainted with the code or working through problems; otherwise, we can just catch up when you're ready with a Pull Request.

**Reply by DaveSkender** [reply]
@totogroo let me know if you want us to keep your name on this one.  I haven’t heard back from you, so I’m not sure if your still interested in doing it. #343

**Reply by DaveSkender** [reply]
ToToGroo ghosted, so if anyone else wants to contribute this one, it's open.

**Comment by DaveSkender** [comment]
Please continue any development related conversation in the tracking Issue #343
